private-templates-service/client/src/AuthenticatedComponent.tsx

```diff
@@ -0,0 +1,34 @@
+import React from "react";
```

Comment:
Should name the file the same as what is exported, in this case requireAuthentication

Reply:
Also, should be in a subfolder or something